base/error.jl

```diff
@@ -50,32 +50,45 @@ macro assert(ex, msgs...)
     :($(esc(ex)) ? $(nothing) : throw(Main.Base.AssertionError($msg)))
 end
 
-# NOTE: Please keep the constant values specified below in sync with the doc string
 const DEFAULT_RETRY_N = 1
 const DEFAULT_RETRY_ON = e->true
 const DEFAULT_RETRY_MAX_DELAY = 10.0
+const DEFAULT_RETRY_FIRST_DELAY = 0.05
+const DEFAULT_RETRY_GROWTH_FACTOR = 5
+const DEFAULT_RETRY_JITTER_FACTOR = 0.1
+const DEFAULT_RETRY_MESSAGE = ""
 
 """
-    retry(f, [retry_on]; n=1, max_delay=10.0) -> Function
+    retry(f, [retry_on]) -> Function
 
-Returns a lambda that retries function `f` up to `n` times in the
-event of an exception. If `retry_on` is a `Type` then retry only
-for exceptions of that type. If `retry_on` is a function
-`test_error(::Exception) -> Bool` then retry only if it is true.
+Returns a lambda that retries function `f` in the event of an exception.  If
```

Comment:
Maybe use "anonymous function" in place of "lambda"? I did a quick search of the Julia docs and this looks like one of two uses of "lambda" to mean anonymous function. I think anonymous function is already used more often, and is easier to guess the meaning of.